packages/prisma-client/src/prisma.ts

```diff
@@ -1,5 +1,5 @@
 import { PrismaClient, Prisma } from "@prisma/client";
-import { PrismaClientKnownRequestError, Decimal } from "@prisma/client/runtime";
+const { PrismaClientKnownRequestError, Decimal } = Prisma;
```

Comment:
Is this also because of `import/no-internal-modules`? It seem like a bad idea to me to use it if it forces us to do this.

Reply:
It forces, but we can make exceptions where possible like we do for the followings:
```js
 allow: [
          "**/server",
          "@lexical/react/*",
          "__generated__/*",
          "react-hot-toast/headless",
 ],
```